src/components/avatar/Avatar.jsx

```diff
@@ -16,7 +16,7 @@ const sizes = {
 const iconSizes = {
   xsmall: 14,
```

Comment:
for now it's same as style names
maybe later we standardise it in all components, same plan is for colors